contracts/auction-handler/FastLaneAuctionHandler.sol

```diff
@@ -94,24 +89,6 @@ contract FastLaneAuctionHandler is FastLaneAuctionHandlerEvents, Ownable, Reentr
 
     uint256 public validatorsTotal;
 
-    uint256 public flStakeSharePayable;
-    uint24 public flStakeShareRatio;
-
-    uint24 public proposalStakeShareRatio;
-    uint256 public proposalDeadline;
-
-    uint256 public minRelayBidAmount = 1 ether; // 1 Matic
-
-    bool public pendingStakeShareUpdate;
-    bool public paused;
-    bool public bid_simulator_enabled = true;
-
-    constructor(uint24 _initialStakeShare, uint256 _minRelayBidAmount) {
-        flStakeShareRatio = _initialStakeShare;
-        minRelayBidAmount = _minRelayBidAmount;
-        emit RelayInitialized(_initialStakeShare, _minRelayBidAmount);
-    }
-
 
     /// @notice Submits a flash bid
     /// @dev Will revert if: already won, minimum bid not respected, not from EOA, or current validator is not participating in PFL.
```

Comment:
```suggestion
    /// @dev Will revert if: already won, minimum bid not respected, or not from EOA
```
Looks like there's no longer any check to see if validators are participating in PFL